src/main/java/pl/antonina/tasks/child/ChildController.java

```diff
@@ -33,8 +36,15 @@ public ChildView getChild(@ApiIgnore Principal parentPrincipal, @PathVariable lo
     }
 
     @PostMapping
-    public void addChild(@ApiIgnore Principal parentPrincipal, @Validated @RequestBody ChildData childData) {
-        childService.addChild(parentPrincipal, childData);
+    public ResponseEntity<Void> addChild(@ApiIgnore Principal parentPrincipal, @Validated @RequestBody ChildData childData) {
```

Comment:
Please check if it works properly. There can be problems when using Void here.

Reply:
I checked and it's working. I don't need to return any body.